app/Service/Student/UpdateStudentProfileService.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Service\Student;
+
+use App\Models\Student;
+use App\Exceptions\{
+    StudentNotFoundException,
+    ResumeNotFoundException
+};
+
+class UpdateStudentProfileService
+{
+    public function execute(string $studentId, array $data): bool
+    {
+        return $this->updateStudentProfile($data, $studentId);
+    }
+
+    public function updateStudentProfile(array $data, string $studentId):bool
+    {
+        $student = $this->getStudentById($studentId);
+        $wasStudentUPdated = $this->updateStudent($data, $student);
+        $wasResumeUpdated = $this->updateResume($data, $student);
```

Comment:
The return of this method should be void as it is a command. Control the Exceptions to show incomplete actions

Reply:
done